dynamic.js

```diff
@@ -0,0 +1,50 @@
+// creating mobile menue elements
```

Comment:
Please add linter config from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) link for html+css+js.

Reply:
> Please add linter config from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) link for html+css+js. It's because you have started to use javaScript in your project.

I already did it in .eslintrc.JSON file xo please check it